### ISSUE TITLE
Multiple input tutorial

### ISSUE THREAD

**Issue by THinnerichs** [pytorch/tutorials#828]
I am currently trying to build a model that takes two different inputs into account, trying to generalize the interaction between both from their properties. 
However, I cannot find any resource on how to build a dataset that allows multiple inputs, while it seems to be quite simple to build the neural net itself. Yet, I haven't found a solution. It would be great to address this issue in the PyTorch documentation, or give a tutorial for this.

**Comment by parmarsuraj99** [comment]
Have you tried Neural Style transfer which takes two images (content and style) and tries to create a combined image.

**Comment by THinnerichs** [comment]
Solution was found in the depths of the torch forums. Thus, this isn't an urgent problem for me anymore. However, as there are a lot of problems concerned with the relation of two different inputs, it'd be great to have a official tutorial for this.

**Comment by parmarsuraj99** [comment]
A tutorial for Style transfer? With perceptual loss?

**Comment by THinnerichs** [comment]
No, just "How to build a PyTorch dataset object that is feeds multiple inputs". Nothing about the problem itself. 

An example problem here is: drug-protein-interaction prediction. Take a representation for the drug and one for the protein, then learn and predict whether there is a interaction between them. Therefor, you need a Dataset that is able to feed forward to two inputs. 

Not everything in machine learning is about CV :)

**Comment by parmarsuraj99** [comment]
That's an interesting problem to work on. Could you please give any link to a data set for the same?

**Comment by THinnerichs** [comment]
[This Github repo](https://github.com/luoyunan/DTINet) is one approach to this problem, and they present their data under ```/data/```. Just try to google/ddg/google-scholar "Drug target interaction prediction" to find many many more approaches and datasets. 

If any more questions arise, feel free to reach out to me.
